source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -691,7 +710,7 @@ static BaseType_t prvGMACInit( NetworkInterface_t * pxInterface )
             NetworkEndPoint_t * pxEndPoint;
```

Comment:
Maybe its more cleaner (or more readable) to have different names for variables (`pxEndPoint`) though they are in different scope.